### DISCUSSION TITLE
Syntax Error: SyntaxError: node_modules\ml-matrix\src\matrix.js: Class private methods are not enabled. Please add `@babel/plugin-transform-private-methods` to your configuration.

### DISCUSSION THREAD

**Discussion by DragonFishYL** [ant-design/ant-design#50963] [Q&A]
Syntax Error: SyntaxError: node_modules\ml-matrix\src\matrix.js: Class private methods are not enabled. Please add `@babel/plugin-transform-private-methods` to your configuration.node14.17.6 ,"antd": "^4.16.13",每次执行npm run都会报这个错误，无法启动服务

**Comment by petercat-assistant** [comment]
这个错误是因为你的 Babel 配置中没有启用对私有方法的支持。你需要在 Babel 配置中添加 `@babel/plugin-transform-private-methods` 插件。以下是解决这个问题的步骤：

1. **安装插件**：
   你需要先安装 `@babel/plugin-transform-private-methods` 插件。运行以下命令：
   ```bash
   npm install --save-dev @babel/plugin-transform-private-methods
   ```

2. **更新 Babel 配置**：
   在你的 Babel 配置文件（例如 `.babelrc` 或 `babel.config.js`）中添加该插件。以下是一个示例配置：

   如果你使用的是 `.babelrc` 文件：
   ```json
   {
     "plugins": ["@babel/plugin-transform-private-methods"]
   }
   ```

   如果你使用的是 `babel.config.js` 文件：
   ```javascript
   module.exports = {
     plugins: ["@babel/plugin-transform-private-methods"]
   };
   ```

3. **重新运行构建**：
   完成上述步骤后，重新运行 `npm run` 命令，看看问题是否解决。

请按照上述步骤操作，如果还有问题，请告诉我。
